druid/src/contexts.rs

```diff
@@ -14,8 +14,8 @@
 
 //! The context types that are passed into various widget methods.
 
+use instant::Instant;
```

Comment:
nit, but generally `std` imports at the top, extern imports below that, internal imports at bottom. 👌